db/migrate/20210316044139_modify_students_table.rb

```diff
@@ -0,0 +1,11 @@
+class ModifyStudentsTable < ActiveRecord::Migration
```

Comment:
@Luke-kb fine to leave is for now.
In general the migration name should be more specific rather then generic.
It should describe what the migration is doing.
```
class RenameStudentsTitle < ActiveRecord::Migration
```